### PR TITLE
Use HTTP Accept-Language to detect locale

### PR DESCRIPTION
**Pull request by Awea**
Hi there have you considered use the HTTP Accept-Language header to detect the locale? Here is a pull request in that way.

### CODE REVIEW

**Comment by ineffyble** [comment]
@Awea are you able to fix the Gemfile order issue raised by codeclimate?

**Comment by Awea** [comment]
@ineffyble Done, sorry for the delay. I don't understand, how codeclimate define the gem order?

**Comment by Awea** [comment]
@ineffyble Oh it's rubocop... ^^

**Comment by Awea** [comment]
The new added gem cause a warning with Rubocop because they were not alphabetically sorted. This doesn't appear using rubocop outside codeclimate :(

**Comment by Gargron** [comment]
I don't care about gem ordering in the Gemfile so don't worry about that one (I wonder if that can be disabled on codeclimate/rubocop)

File: app/controllers/concerns/localized.rb [hunk 1]
```diff
@@ -26,6 +26,6 @@ def set_locale
   end
 
   def default_locale
-    ENV.fetch('DEFAULT_LOCALE') { I18n.default_locale }
+    ENV.fetch('DEFAULT_LOCALE') { http_accept_language.compatible_language_from(I18n.available_locales) }
```
Comment (Gargron) [inline app/controllers/concerns/localized.rb]: What happens when there is no accept-language header at all? I don't see a fallback to `I18n.default_locale` in the block.

Comment (Awea) [inline app/controllers/concerns/localized.rb]: You're right. I updated it.

**Review by Gargron — APPROVED** [review]

**Pull request merged**